usr/share/rear/verify/GALAXY11/default/400_start_galaxy_daemons.sh

```diff
@@ -0,0 +1,16 @@
+#
+# start commvault daemons and check that they run and connect to the backup server
+#
+#
+#
+
+# Commvault dies when it finds /lib/tls with libpthread
+# Relax-and-Recover has a symlink from /lib to /lib/tls and I don't remember why (Schlomo)
+
+# delete problematic symlink
+rm -f /lib/tls
+
+Galaxy start
```

Comment:
Only an offhanded hint
(I know nothing about the 'Galaxy' command):

Sometimes a service/daemon starting command returns
as soon as it succeeds to get the daemon started
but this may not mean the daemon is already ready to operate
so a direct subsequent command that tests if the daemon is
accessible and ready to be normally used by users may fail
but a few moments/seconds later it would "just work".

So perhaps a hardcoded 'sleep 1' or similar may be right
or some limited retry loop for 'Galaxy list'?